apps/monitor-client/lib/index.tsx

```diff
@@ -6,15 +6,20 @@ import { AppProvider } from './context';
 import { ErrorBoundaryComponent } from './components/ErrorBoundary';
 import './index.css';
 
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+const queryClient = new QueryClient();
```

Comment:
lets move queryClient to the separate file (./lib/queryClient/index.ts) and pass global config to play with it:
export const queryClient = new QueryClient({
  defaultOptions: {
    queries: {
      staleTime: 60 * 1000 // 1 minutes,
      cacheTime: 10 * 60 * 1000 // 10 minutes
    }
  }
})